doc/nrf/libraries/others/trusted_storage.rst

```diff
@@ -0,0 +1,126 @@
+.. trusted_storage_readme:
+
+Trusted storage library
+#######################
+
+.. contents::
+   :local:
+   :depth: 2
+
+Overview
+********
+
+The trusted storage library allows its users to securely store data.
+It is based on PSA Certified Secure Storage API and provides data protection through integrity and confidentiality using AEAD algorithms.
+
+Interfaces
+==========
+
+This trusted storage library provides two storage interfaces for use with device-protected storage:
+
+* Internal trusted storage, designed to store critcal data that must be placed inside internal flash memory.
+  Examples of assets that require this kind of storage are replay protection values for external storage and keys for use by components of the PSA Root of Trust.
+* Protected storage, designed to store all other critical data that do not need to be stored inside internal flash memory.
+
+Backends
+========
+
+The following backends are available in the trusted storage library:
+
+SECURE_STORAGE_BACKEND_AEAD
```

Comment:
Are we changing all instances of SECURE_STORAGE to TRUSTED_STORAGE now?

Reply:
I renamed to to encrypted storage, in #13833